<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfDialog</class>
 <widget class="QDialog" name="ConfDialog">
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="fontsTab">
      <attribute name="title">
       <string>Fonts &amp;&amp; Colors</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Proportional</string>
         </property>
         <property name="buddy">
          <cstring>mainFontBox</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFontComboBox" name="mainFontBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mainFontSizeSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string notr="true">pt</string>
           </property>
           <property name="minimum">
            <number>6</number>
           </property>
           <property name="maximum">
            <number>72</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Fi&amp;xed-width</string>
         </property>
         <property name="buddy">
          <cstring>fixedFontBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QFontComboBox" name="fixedFontBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="fontFilters">
            <set>QFontComboBox::MonospacedFonts</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="fixedFontSizeSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string notr="true">pt</string>
           </property>
           <property name="minimum">
            <number>6</number>
           </property>
           <property name="maximum">
            <number>72</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&amp;Scrollback</string>
         </property>
         <property name="buddy">
          <cstring>scrollbackFontBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QFontComboBox" name="scrollbackFontBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="scrollbackFontSizeSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string notr="true">pt</string>
           </property>
           <property name="minimum">
            <number>6</number>
           </property>
           <property name="maximum">
            <number>72</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Colors</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Main &amp;Text</string>
              </property>
              <property name="buddy">
               <cstring>mainTextColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="KColorButton" name="mainTextColorButton"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Main &amp;Background</string>
              </property>
              <property name="buddy">
               <cstring>mainBgColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="KColorButton" name="mainBgColorButton"/>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="customFsMarginColorCheckBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>&lt;p&gt;Disables automatic margin color detection when in fullscreen mode.&lt;/p&gt;</string>
              </property>
              <property name="text">
               <string>Fu&amp;llscreen Margins</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="KColorButton" name="fsMarginColorButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Status T&amp;ext</string>
              </property>
              <property name="buddy">
               <cstring>bannerTextColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="KColorButton" name="bannerTextColorButton"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Status B&amp;ackground</string>
              </property>
              <property name="buddy">
               <cstring>bannerBgColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="KColorButton" name="bannerBgColorButton"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mediaTab">
      <attribute name="title">
       <string>Media</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout_6">
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="allowMusicCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Enable &amp;Music</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QCheckBox" name="muteSoundCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>&lt;p&gt;Mutes all sound when the application window loses focus.&lt;/p&gt;</string>
           </property>
           <property name="text">
            <string>Mute Sound in &amp;Background</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QCheckBox" name="allowSoundEffectsCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Enable Sound &amp;Effects</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Vo&amp;lume</string>
           </property>
           <property name="buddy">
            <cstring>volumeSlider</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QSlider" name="volumeSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="sliderPosition">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QFormLayout" name="formLayout_7">
         <item row="1" column="0" colspan="2">
          <widget class="QCheckBox" name="allowVideoCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Enable &amp;Video</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QCheckBox" name="smoothScalingCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>&lt;p&gt;Applies a bilinear filter when scaling images in order to avoid scaling artifacts (pixelization and distortion.)&lt;/p&gt;</string>
           </property>
           <property name="text">
            <string>Use Smooth Image &amp;Scaling</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="allowGraphicsCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Enable &amp;Graphics</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="miscTab">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="marginSizeLabel">
           <property name="text">
            <string>&amp;Margin Size</string>
           </property>
           <property name="buddy">
            <cstring>marginSizeSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="marginSizeSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string>px</string>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&amp;Fullscreen Width</string>
           </property>
           <property name="buddy">
            <cstring>fullscreenWidthSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="fullscreenWidthSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="scriptWrapLabel">
           <property name="text">
            <string>&amp;Scriptfile Wrap</string>
           </property>
           <property name="buddy">
            <cstring>scriptWrapSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="scriptWrapSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="specialValueText">
            <string>None</string>
           </property>
           <property name="suffix">
            <string>ch</string>
           </property>
           <property name="minimum">
            <number>20</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QCheckBox" name="smartFormattingCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>&lt;p&gt;Converts ASCII quotes and dashes to typographical ones.&lt;/p&gt;</string>
           </property>
           <property name="text">
            <string>Sma&amp;rt Formatting</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QCheckBox" name="overlayScrollbackCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>&lt;p&gt;Make the scrollback window the same size as the game window and position it exactly over it.&lt;/p&gt;</string>
           </property>
           <property name="text">
            <string>&amp;Overlay Scrollback</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QCheckBox" name="softScrollCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>&lt;p&gt;Scrolls-in new text line by line instead of all at once.&lt;/p&gt;</string>
           </property>
           <property name="text">
            <string>Soft &amp;Text Scrolling</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>199</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>mainFontBox</tabstop>
  <tabstop>mainFontSizeSpinBox</tabstop>
  <tabstop>fixedFontBox</tabstop>
  <tabstop>fixedFontSizeSpinBox</tabstop>
  <tabstop>scrollbackFontBox</tabstop>
  <tabstop>scrollbackFontSizeSpinBox</tabstop>
  <tabstop>mainTextColorButton</tabstop>
  <tabstop>mainBgColorButton</tabstop>
  <tabstop>customFsMarginColorCheckBox</tabstop>
  <tabstop>fsMarginColorButton</tabstop>
  <tabstop>bannerTextColorButton</tabstop>
  <tabstop>bannerBgColorButton</tabstop>
  <tabstop>allowMusicCheckBox</tabstop>
  <tabstop>allowSoundEffectsCheckBox</tabstop>
  <tabstop>muteSoundCheckBox</tabstop>
  <tabstop>volumeSlider</tabstop>
  <tabstop>allowGraphicsCheckBox</tabstop>
  <tabstop>allowVideoCheckBox</tabstop>
  <tabstop>smoothScalingCheckBox</tabstop>
  <tabstop>marginSizeSpinBox</tabstop>
  <tabstop>fullscreenWidthSpinBox</tabstop>
  <tabstop>scriptWrapSpinBox</tabstop>
  <tabstop>softScrollCheckBox</tabstop>
  <tabstop>smartFormattingCheckBox</tabstop>
  <tabstop>overlayScrollbackCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
